snakes_cafe.py

```diff
@@ -2,30 +2,78 @@
 import csv
 import uuid
 
-menus = {'Appetizers': {('Wings', 2.00): 0, ('Cookies', 15.00): 0,
-         ('Spring Rolls', 4.00): 0, ('Rings', 4.00): 0,
-         ('Shrimp bites', 4.00): 0, ('Wontons', 4.00): 0,
-         ('Crab Dip', 4.00): 0, ('Sliders', 4.00): 0,
-         ('Clams', 8.00): 0},
-         'Entrees': {('Salmon', 5.00): 0, ('Steak', 6.00): 0,
-                     ('Meat Tornado', 4.00): 0, ('A Literal Garden', 3.00): 0,
-                     ('Pasta', 4.00): 0, ('Ribs', 4.00): 0,
-                     ('Cabbage Rolls', 8.00): 0, ('Pizza', 4.00): 0,
-                     ('Paella', 4.00): 0},
-         'Desserts': {('Ice Cream', 500.0): 0, ('Cake', 30.00): 0,
-                      ('Pie', 3.00): 0, ('Pudding', 4.00): 0,
-                      ('Fruit', 4.00): 0, ('Sorbet', 4.00): 0,
-                      ('Torte', 4.00): 0, ('Flan', 4.00): 0,
-                      ('Apple', 4.00): 0},
-         'Drinks': {('Coffee', 4.00): 0, ('Tea', 3.00): 0,
-                    ('Blood of the Innocent', 6.00): 0, ('Pop', 4.00): 0,
-                    ('Wine', 4.00): 0, ('Beer', 4.00): 0,
-                    ('Sake', 4.00): 0, ('Cocoa', 4.00): 0,
-                    ('Evian', 4.00): 0},
-         'Sides': {('Frys', 4.00): 0, ('Salad', 3.00): 0,
-                   ('Bread', 6.00): 0, ('Slaw', 4.00): 0, ('Soup', 4.00): 0,
-                   ('Rice', 4.00): 0, ('Spinach', 4.00): 0,
-                   ('Sauce', 4.00): 0, ('Crab dip', 4.00): 0}}
+
+class Order:
+    """ Class for orders """
+
+
+def __init__(self):
```

Comment:
All of the these method definitions need to be indented in order to be included in the class. Currently they are just functions defined after the class.